tests/fixtures/themes/test/pages/no-soft-component-class.htm

```diff
@@ -0,0 +1,5 @@
+url = "/no-soft-component-class"
+
+[@PeterPan\Nevernever\Land noComponentExist]
```

Comment:
Is there a test for a component that does exist being included as a soft component?